src/components/NewMessageModal.vue

```diff
@@ -294,6 +294,51 @@ export default {
 		},
 		toHtml,
 		plain,
+		async onNewSentMailbox(data, account) {
+			showWarning(t('mail', 'Setting Sent default folder'))
+			let newSentMailboxId = null
+			const mailboxes = this.mainStore.getMailboxes(data.accountId)
+			const sentMailboxId = mailboxes.find((mailbox) => mailbox.name === account.personalNamespace + 'Sent' || mailbox.name === account.personalNamespace + t('mail', 'Sent'))?.databaseId
```

Comment:
nit. first look for the localized, then search for `Sent`. this ensures that the localized takes priority.